README.rst

```diff
@@ -13,8 +13,9 @@ ExoCTK is an open-source, modular data analysis package focused primarily on atm
 * Integrations and Groups Calculator
 * Transit Light-Curve Fitter
 * Limb Darkening Calculator
-* Atmopsheric Retrievals
-* Atmospheric Forward Modeling - Currently only available through the `website <https://exoctk.stsci.edu/fortney>`_
+* Atmospheric Retrievals
+* Phase Constraint Calculator
+* Atmospheric Forward Modeling - Currently only available through the `website <https://exoctk.stsci.edu/fortney>`_.
 
 For more information on each package visit our documentation `website <https://exoctk.readthedocs.io/en/latest/>`_.
```

Comment:
Let's have this say "our documentation at readthedocs" -- so it doesn't say website a bunch of times.